### PR TITLE
Changes to mIF schema and tests to accommodate MDA data

### PR DESCRIPTION
**Pull request by scvannost**

### CODE REVIEW

File: cidc_schemas/schemas/assays/components/imaging/mif_roi.json [hunk 1]
```diff
@@ -25,11 +27,11 @@
   },
   "mergeStrategy": "objectMerge",
   "required": [
+    "roi_id",
```
Comment (roshni-b) [inline cidc_schemas/schemas/assays/components/imaging/mif_roi.json:30]:
```suggestion
    "roi_id",
```
```suggestion
    "roi_id",
    "tissue_seg_data",
    "tissue_seg_data_summary"
```

**Review by roshni-b — CHANGES_REQUESTED** [review]
`tissue_seg_data` and `tissue_seg_data_summary` should be "required properties". 
And don't forget to bump the schemas package version, and regenerate documentation.

File: tests/test_assays.py [hunk 2]
```diff
@@ -385,6 +385,7 @@ def test_mif():
         "files": {
             "regions_of_interest": [
                 {
+                    "roi_id": "foo",
```
Comment (roshni-b) [inline tests/test_assays.py:388]:
```suggestion
                    "roi_id": "foo",
                    "tissue_seg_data": text,
                    "tissue_seg_data_summary": text,
```

File: cidc_schemas/schemas/assays/components/imaging/mif_roi.json (hunk 1 above)
Comment (scvannost) [reply cidc_schemas/schemas/assays/components/imaging/mif_roi.json:30]: my understanding is that while we should expect it from MDA, DFCI it set up to process it either with or without, but idk how they choose that. Jason seemed to imply that we shouldn't expect it from them

Comment (roshni-b) [reply cidc_schemas/schemas/assays/components/imaging/mif_roi.json:30]: Makes sense!

**Review by roshni-b — APPROVED** [review]

**Pull request merged**